JumpRoyale/src/GameObjects/BaseObject.cs

```diff
@@ -0,0 +1,5 @@
+using Godot;
+
+#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
+
+public record BaseObject(Vector2I Point, Vector2I Left, Vector2I Middle, Vector2I Right) { }
```

Comment:
Please add a documentation comment indicating what the arguments represent.

Reply:
Added 👍 I guess the `Point` introduced the most confusion